lib/windows/app/index.js

```diff
@@ -61,6 +61,18 @@ Module._load = function load(path) { // eslint-disable-line no-underscore-dangle
             return ReactDOM;
         case 'react-redux':
             return ReactRedux;
+        case 'nrfconnect/logger':
+            return api.logger;
+        case 'nrfconnect/pc-ble-driver-js':
+            return api.bleDriver;
```

Comment:
This is a good point. The cleanest would be change pc-ble-driver-js to export api members at toplevel, but I guess that's not a choice now.